package.json

```diff
@@ -4,8 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "generate-fake-data": "babel-node scripts/generateTestData  --presets es2015",
     "build": "mkdir -p out && cp data/impactData.js out/impactData.js && cp data/reportInfo.js out/reportInfo.js && webpack",
-    "watch": "mkdir -p out && cp data/impactData.js out/impactData.js && cp data/reportInfo.js out/reportInfo.js && webpack --watch",
+    "build-dev-watch": "mkdir -p out && cp data/test/impactData.js out/impactData.js && cp data/test/reportInfo.js out/reportInfo.js && webpack --watch",
+    "build-dev": "mkdir -p out && cp data/test/impactData.js out/impactData.js && cp data/test/reportInfo.js out/reportInfo.js && webpack",
+    "webpack": "webpack",
```

Comment:
Found myself wanting to sometimes just run `webpack` to recompile the js without needing to copy over `impactData` again, so added this command.